### ISSUE TITLE
The loss of text-adda in train_tgt doesn't descend

### ISSUE THREAD

**Issue by Originofamonia** [bzantium/bzantium#1]
Hi,

I encountered a problem in reproducing the results in your code in [https://github.com/deep0learning/text-adda](url). I found the loss of `train_tgt` "training encoder for target domain" is not descending. I have tried many different lr and betas for tgt_encoder and critic, but still can't get it to work. Could you please release your hyperparameters for the train_tgt functions?

Thanks!

**Comment by bzantium** [comment]
Thank you for your interest. text-adda is old version of [bert-ADD](https://github.com/bzantium/bert-AAD) repo so I recommend you to try this. If you still encounter the same problem, please ask me again! Thanks.
